client/state/sites/reducer.js

```diff
@@ -73,7 +73,15 @@ export function items( state = null, action ) {
 		case SITE_RECEIVE:
 		case SITES_RECEIVE:
 			// Normalize incoming site(s) to array
-			const sites = action.site ? [ action.site ] : action.sites;
+			const maybeUnmanageableSites = action.site ? [ action.site ] : action.sites;
+			// filter out anything the current user can't manage
+			const sites = maybeUnmanageableSites.filter( site => {
+				return site && site.capabilities;
```

Comment:
we can lose the `return` here. Or we can `import { has } from 'lodash/fp` and just:
```js
const sites = maybeUnmanageableSites.filter( has( 'capabilities' ) );
```

Reply:
Good point that we don't need a variable here. We're okay here with a simple truthy check, but I added the `!!` to note that we are interested in a boolean vs, trying to map over this. Updated this to a single line in b0ebdfb